tests/test_utils.py

```diff
@@ -235,7 +235,7 @@ def test_get_os_platform_linux(tmp_path):
     with patch('platform.machine', return_value='x86_64'):
         with patch('platform.system', return_value='Linux'):
             os_platform = get_os_platform(filepath)
-    assert os_platform.system == "Ubuntu"
+    assert os_platform.system == "ubuntu"
```

Comment:
We could also add a test using core 20 os-release values.

Reply:
Unsure of provided value, but added anyway.